src/components/Intercom/intercomChecklist.ts

```diff
@@ -0,0 +1,24 @@
+export const enum IntercomChecklistEvents {
+  "CreatedComponent" = "CreatedComponent",
+  "LinkedComponentToConfig" = "LinkedComponentToConfig",
+  "LinkComponentToIncident" = "LinkComponentToIncident",
+  "CreatedIncident" = "CreatedIncident",
+  "AddedResponderToIncident" = "AddedResponderToIncident",
+  "AddedDoDToIncident" = "AddedDoDToIncident",
+  "LinkedConfigToIncident" = "LinkedConfigToIncident",
+  "LinkedLogToIncident" = "LinkedLogToIncident",
+  "LinkedHealthCheckToIncident" = "AddHealthCheckToIncident",
+  "CreatedHealthCheck" = "CreatedHealthCheck",
+  "CreatedConfigScrapper" = "CreatedConfigScrapper",
+  "CreatedLogBackend" = "CreatedLogBackend"
+}
+
+// Investigate if we can use this to track the user
+export function sendIntercomChecklistEvent(
```

Comment:
Move this to services/analytics.js and call the function, sendAnalyticEvent - we don't wont intercom spread all over the app

+ if no intercom appi id specified this should be a no-op

Reply:
The window.Intercom function should only be available when the intercom has been initialized, which only happens when the APP_ID is available.